--- orig/src/simple_stats.h
+++ tidy/src/simple_stats.h
@@ -19,9 +19,11 @@
 #ifndef SIMPLE_STATS_H
 #define SIMPLE_STATS_H
 
+/*
 #ifdef __cplusplus
 extern "C" {
 #endif
+*/
 
 #include <stdio.h>
 
@@ -54,8 +56,10 @@
 
 const char *simple_stats_version();
 
+/*
 #ifdef __cplusplus
 }
 #endif
+*/
 
 #endif /* SIMPLE_STATS_H */
--- orig/tests/echeck.h
+++ tidy/tests/echeck.h
@@ -19,9 +19,11 @@
 #ifndef ECHECK_H
 #define ECHECK_H
 
+/*
 #ifdef __cplusplus
 extern "C" {
 #endif
+*/
 
 #include <stdlib.h>		/* for size_t */
 #include <stdio.h>		/* for FILE */
@@ -281,8 +283,10 @@
 #define check_status(val)\
 	echeck_status_m(stderr, ECHECK_FUNC, __FILE__, __LINE__, val, NULL)
 
+/*
 #ifdef __cplusplus
 }
 #endif
+*/
 
 #endif /* ECHECK_H */
